Dolphin scripts/Entrance Randomizer/lib/entrance_rando.py

```diff
@@ -94,15 +98,15 @@ def highjack_transition_rando():
 def link_two_levels(first: Area, second: Area):
     first.con_left -= 1
     second.con_left -= 1
-    return [first, second]
+    return (first, second)
 
 
 def unlink_two_levels(first: Area, second: Area):
     first.con_left += 1
     second.con_left += 1
 
 
-def connect_to_existing(level_list: list[Area], index: int, link_list: list[list[Area]]):
+def connect_to_existing(level_list: Sequence[Area], index: int, link_list: list[tuple[Area, Area]]):
     global total_con_left
     total_con_left += level_list[index].con_left
     levels_available = []
```

Comment:
That'll resolve most your `Unknown` typing issues in this function

```suggestion
    levels_available: list[Area] = []
```

Reply:
Will be changed in next commit.